meter/config.go

```diff
@@ -28,6 +30,38 @@ var registry meterRegistry = make(map[string]func(map[string]interface{}) (api.M
 
 // NewFromConfig creates meter from configuration
 func NewFromConfig(typ string, other map[string]interface{}) (v api.Meter, err error) {
+	if typ == "custom" && other["public"] != nil {
```

Comment:
Die Logik ist an der falschen Stelle. Besser wäre es, das vorgenerierte Yaml mit seinem Namen als normalen Charger etc zu registrieren und hier nur noch zu konfigurieren (=hier ändert sich nichts). zur Registrierung über alle Templates drüber gehen und schauen ob die sich für Registrierung eignen und per `registry.Add` hinzufügen.

Reply:
@andig Den Bezug über `public` herzustellen war mein letzter Stand aus unserer Diskussion, und das wir war erst mal ein Weg wich das ganze auch mal end-to-end überhaupt zum Laufen bekommen habe. Worüber ich schon mal sehr froh bin. Hat bis hierhin auch schon ewig gedauert und immer wieder zu viel Frust geführt.

Ich kann mal schauen wie ich das dahingehend erweitern kann. Wird evtl. ne Weile dauern.

Reply:
So eine Logik hätte zur Folge dass (fast) jedes Setup das über den CLI gemacht würde für jede Komponente darüber gehen würde. Denn der CLI kann nur Konfigurationen verarbeiten welche eine `params` Struktur definiert haben.

Ich habe aber leider noch keine Idee wo man so etwas in EVCC einbinden muss/sollte.